config/leka_os_version

```diff
@@ -0,0 +1 @@
+1.1.0
```

Comment:
I'm wondering if `os_version` is not better, than `leka_os_version` especially if we change the name of the variable in the Makefile